cli/bin/grain.js

```diff
@@ -193,6 +200,12 @@ program
 program
   .command("lsp <file>")
   .description("check a grain file for LSP")
+  .forwardOption(
+    "-S, --stdlib <path>",
+    "override the standard libary with your own",
+    null,
+    stdlibPath
+  )
```

Comment:
If these are specified separately from the main program, then we need to add others. such as `--output`. I'm not sure all of them that'll need to be duplicated

Reply:
Yup yup. Having a realization now. I'll make these inherit all of the compiler config for now until we get it truly separated.